src/Prod/README.md

```diff
@@ -0,0 +1,122 @@
+# Prod
+```haskell
+Prod Number
+```
+`Prod` is a `Monoid` that will combine two numbers using Multiplication.
+
+```js
+const Prod = require('crocks/Prod')
+const mconcatMap = require('crocks/helpers/mconcat')
+
+const emptyProd = Prod.empty()
+const tenProd = Prod(10)
+const hundredProd = Prod(100)
+
+emptyProd
+  .concat(emptyProd)
+//=> Prod 1
+
+tenProd
+  .concat(hundredProd)
+//=> Prod 1000
+
+// prodBy2 :: Number -> Prod
+const prodBy2 =
+  mconcatMap(Prod, 2)
```

Comment:
`mconcatMap` is defined as `mconcatMap :: Monoid m => m -> (b -> a) -> ([ b ] | List b) -> m a` so this code will throw as `2` is not a function. Also the third takes a `Foldable` structure like `List` or `Array`.

You may wanna redo these example to either use the common signature for `mconcatMap` or abandon it and do what I think you were going for:
```js
const double =
  compose(concat(Prod(2)), Prod) 
```
Personally when I write examples for documentation, I will try it in some sort of console or sandbox like [evil-playground](https://github.com/evilsoft/evil-playground). That way I can ensure that the example I am providing runs without error.

Reply:
Thanks for the advice! I did install the library locally to play with, but I could not figure out many good examples.  Excuses are excuses, I did commit bad code.

Honestly, I don't know what happened here; it looks like my `Sum` example was committed fine, but this one was not.  Shame shame shame.